src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const styleArray = sourceString.split(';');
+  const removingSpaces = styleArray.map((lines) => lines.trim());
+  const filterValues = removingSpaces.filter((elem) => elem !== '');
```

Comment:
You can chain the `split`, `map`, and `filter` methods to make the code more readable and concise. Chaining these methods will also adhere to the good practice of not using unnecessary intermediate variables like `styleArray` and `removingSpaces`.